File: Conditionals/Time Notes.py
#Sasha Rinitra, How to et the time of the day

import time
#first instance of time in programming
#print(time.gmtime())

#current time in seconds since gmtime
current = time.time()

#Current time as we are used to seeing it
now = time.ctime(current)
print(now)

#get just the hour
local_time = time.localtime(current)
hours = local_time.tm_hour
minutes = local_time.tm_min
print(hours)

File: Conditionals/Time_Notes.c
// Sasha Rinitra, Getting time in C
#include <stdio.h>
#include <time.h>

int main(void) {
    // time since Jan 1, 1970
  time_t seconds;

  seconds = time(NULL);
  //printf("Seconds since January 1, 1970 = %d\n", seconds);

// Current time
  time_t rawtime;
  struct tm * timeinfo;

  time(&rawtime);
  timeinfo = localtime(&rawtime);
  printf("Our curent time and date is %s\n", asctime(timeinfo));

  //current hour
  time_t now = time(NULL);
  struct tm *tm_struct = localtime(&now);
  int hour = tm_struct->tm_hour;
  printf("%d\n", hour);

  return 0;
}